config/initializers/ruby_money.rb

```diff
@@ -1,8 +1,16 @@
 # frozen_string_literal: true
 
+require 'money/bank/currencylayer_bank'
+
 Money.locale_backend = :i18n
 
-eu_bank = EuCentralBank.new
-Money.default_bank = eu_bank
+mclb = Money::Bank::CurrencylayerBank.new
+mclb.access_key = ENV.fetch("CURRENCY_LAYER_API_KEY", "")
+mclb.currencylayer = true
```

Comment:
What does this flag do and why is it needed? From my perspective as an outside reviewer who hasn't done any research on CurrencyLayer, this line of code reads like "Oh dear CurrencyLayer gem, which gives me a class called `CurrencylayerBank`, please _really_ **really** access CurrencyLayer by the way!".